frontend/assessments/2/public/script.js

```diff
@@ -0,0 +1,114 @@
+function handleClick() {
+    const input = document.getElementById("input");
+    const button = document.getElementById("submit");
+    const messages = document.querySelector(".messages");
+    
+    // Get current date and time
+    const currentDate = new Date();
+    const dateTimeString = currentDate.toLocaleString();
+
+    // Create a new div element
+    const element = document.createElement("div");
+    element.className = "transactions";
+
+    // Add content to the div
+    element.innerHTML = `<span style="color: green;">SELL</span>${input.value} Stocks<br>${dateTimeString}`;
+
+    element.style.border = "1px solid";
+    element.style.padding = "5px";
+    element.style.width = "95%";
+    element.style.height = "100%";
```

Comment:
lets put html in index file and manipulate it using display & visibility properties.